ext/systemd/puppetmaster.service

```diff
@@ -5,7 +5,8 @@ After=basic.target network.target
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetmaster
-ExecStart=/usr/bin/puppet master $PUPPETMASTER_EXTRA_OPTS --no-daemonize
+EnvironmentFile=-/etc/default/puppetmaster
+ExecStart=/opt/puppetlabs/puppet/bin/puppet master $PUPPETMASTER_EXTRA_OPTS --no-daemonize
```

Comment:
So, I noticed we also have the puppetmaster.service file, which is referenced in the specfile in the puppet repo, as well as the puppet.service file, but no where in puppet-agent. I'm not sure if we even need this file, but it should probably be updated according to the new layout